lib/external_service_new/zendesk_downloader.rb

```diff
@@ -0,0 +1,51 @@
+module ExternalServiceNew
+	class ZendeskDownloader
+		def initialize(dispatcher:, logger:nil)
+			@dispatcher = dispatcher
+			@logger			= logger || respond_to?(:logger) ? logger : nil
+		end
+
+		def each_list(&block)
+			return to_enum(:each_list) unless block
+			[[
```

Comment:
It for the `each` to treat an array of array. Otherwise it will check every value one by one. Should i use a hash?